packages/contracts-bedrock/test/dispute/DisputeGameFactory.t.sol

```diff
@@ -350,6 +350,10 @@ contract FakeClone {
         return hex"FF0420";
     }
 
+    function parentHash() external pure returns (bytes32) {
```

Comment:
this isn't needed is it?

Reply:
Not necessarily - just put it there for completion. These functions aren't used.